packages/react-native/src/types/Module.ts

```diff
@@ -184,6 +184,31 @@ export interface Module {
    */
   displayNotification(notification: Notification): Promise<string>;
 
+  /**
+   * API used to check whether you can create trigger notification with exact timestamp on Android >= 12.
```

Comment:
I noticed that you use Android version in one spot and Android API number in another, perhaps best to specify both in both locations? Just in case - react-native devs don't always know everything about android versioning

```suggestion
   * API used to check whether you can create trigger notification with exact timestamp on Android >= 12 / API >= 31
```